src/locales/ja-JP.yml

```
@@ -0,0 +1,47 @@
+commands:
+  novelai:
+    description: AI で絵を描く
+    usage: |-
+      コンマで区切られた英語の生成呪文 (プロンプト) を入力してください。例：1girl, red eyes, black hair。
+      モデルに用いられるタグは Danbooru とほとんど同じです。
+      興味があったら、レポジトリにスターを付けてください：https://github.com/koishijs/novelai-bot
+
+    options:
+      enhance: 向上 (enhance) モードを有効
+      model: モデルを指定
+      resolution: 画像解像度を設定
+      sampler: サンプラーを指定
+      seed: シード値を設定
+      steps: ステップ数を設定
+      scale: CFG スケール値を設定
+      strength: ノイズ除去強度を設定
+      noise: ノイズ強度を設定
+      undesired: 反対呪文 (ネガティブプロンプト) を設定
+
+    messages:
+      expect-prompt: 生成呪文を入力してください。
+      expect-image: 画像を入力してください。
+      latin-only: 英数字だけが入力可能です。
+      too-many-words: 入力したタグが多すぎる。
+      forbidden-word: 一部の入力したタグが禁止されている。
+      concurrent-jobs@random:
+        - 後でね～今、猫の手も借りたいなの！
+        - 描けるの、た、タブレットが起動できませんだから。
+        - じゃ、まず絵を教えて。
+      waiting@random:
+        - 私はプロ絵師だから、どんな絵でも描けるの。
+        - 仕事している…
+      pending: 仕事している前に {0} つの絵が完遂するべきです。
+      invalid-size: 向上モードは AI 生成画像のみに用いられる。img2img (指定画像から生成) を使いたければ、「--enhance」を追加せずにコマンドを再実行してください。
+      invalid-resolution: 無効な解像度。幅と高さが64の倍数である必要があります。
```

```suggestion
      invalid-resolution: 無効な解像度。幅と高さが 64 の倍数である必要があります。
```

Should add spaces between number and kana?